### DISCUSSION TITLE
Open a second window from the application menu

### DISCUSSION THREAD

**Discussion by theoreaves** [NativePHP/laravel#306] [Q&A] [answered]
How can I open a new window to a particular route from the Application Menu?

**Comment by simonhamp** [comment, marked as answer]
I believe at the moment you'd need to use an '[Event-based menu item](https://nativephp.com/docs/1/the-basics/application-menu#event-based-menu-items)' and have the logic for opening a new window in the listener, though I haven't tried this.

Honestly, I think creating an event and a listener for this is a lot of work. I'd love to see a more ergonomic/idiomatic approach.

Let us know how you get on!

**Reply by theoreaves** [reply]
For now, I've decided to take a different approach.  I've put my app in the menubar, so I can open windows as needed from there.

**Reply by Tray2** [reply]
It works using the event approach that @simonhamp described

I've used it in the project I'm building, before making them modals instead.
